experiment/template/test_eVOLVER.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
```

Comment:
Could you explain exactly what this is testing? It's replicating the main script, which we might want to avoid by using a common utility file or something for doing all of these setup/directory creation steps, I'm worried about the duplicated code for option parsing and whatnot. 

Maybe we should talk about implementing a real testing library/infrastructure. I wrote a little junk script to do a semi fuzzing type test, but I think if we want to start doing tests we should really think about it so they are easy to maintain and expand going forward.